bin/stafftools

```diff
@@ -37,6 +37,36 @@ class JiraStaffTools < Thor
     output_json(response.body)
   end
 
+  desc "current <git_hub_installation_id>", "Display the current sync item"
```

Comment:
I wonder if theres a way to make this `status current` to indicate its related to status?

Reply:
done, you can now invoke it with `bin/stafftools status 1231231 --current` (or leave `--current` off to get the original status behavior)